packages/react-components/source/react/library/icon/icons.js

```diff
@@ -579,6 +579,16 @@ const icons = {
     ),
   },
 
+  'empty-circle': {
```

Comment:
Wondering if this should be `circle-empty` instead, to correspond to the pattern established by `star-empty` (e.g. like if we had a `circle-filled` in the future). Thoughts?

Reply:
Ah yeah I was naming it based on `check-circle`, `info-circle`, `minus-circle` since that's kinda how we were using it in our app, but I think your idea aligns more with the other icons.

Reply:
Oh gotcha. Yeah, maybe the existing ones could have been more consistent.